content-src/components/DiscoveryStreamComponents/List/_List.scss

```diff
@@ -162,6 +162,7 @@ $item-line-height: 20;
 
     display: flex;
     justify-content: space-between;
+    height: 100%;
```

Comment:
Is this necessary?

Reply:
yes, without this link doesn't take full height from parent and fails to bottom align source.